jsonify.js

```diff
@@ -27,8 +26,7 @@ var lf_processArguments = function(){
 
     while (0 < l_aryArgs.length){
         l_strFlag = l_aryArgs.shift();
```

Comment:
I would probably get rid of the 'flag' from the variable name, and just call it a straight up argument.